install_config/persistent_storage/persistent_storage_manila.adoc

```diff
@@ -0,0 +1,265 @@
+the provisioner will create for the volume.
+<2> Set of Manila availability zones that the volume might be created in.
+
+Configure the provisioner to connect, authenticate, and authorize to the Manila service by using
+environment variables. Select the right combination of environment variables for your installation
+from the following list:
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_NAME
+OS_TENANT_NAME
+----
+
+[source]
+----
+OS_USERID
+OS_PASSWORD
+OS_AUTH_URL
+OS_TENANT_ID
+----
+
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_ID
+OS_TENANT_NAME
+----
+
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_ID
+OS_TENANT_ID
+----
+
+To pass the variables to the provisioner, use a `Secret`.
+Example of the `Secret` for the first variables combination:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: manila-provisioner-env
+type: Opaque
+data:
+  os_username: <base64 encoded Manila username>
+  os_password: <base64 encoded password>
+  os_auth_url: <base64 encoded OpenStack Keystone URL>
+  os_domain_name: <base64 encoded Manila service Domain>
+  os_tenant_name: <base64 encoded Manila service Tenant/Project name>
+----
+
+[NOTE]
+====
+Newer OpenStack versions use "project" instead of "tenant," however, the
+environment variables used by the provisioner must use `TENANT` in their
+names.
+====
+
+The last step is to start the provisioner itself, for example, using a deployment:
+
+[source, yaml]
+----
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: manila-provisioner
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: manila-provisioner
+    spec:
+      serviceAccountName: manila-provisioner-runner
+      containers:
+        - image: "registry.access.redhat.com:8888/openshift3/manila-provisioner:latest"
+          imagePullPolicy: "IfNotPresent"
+          name: manila-provisioner
+          env:
+            - name: "OS_USERNAME"
+              valueFrom:
+                secretKeyRef:
+                  name: manila-provisioner-env
+                  key: os_username
+            - name: "OS_PASSWORD"
+              valueFrom:
+                secretKeyRef:
+                  name: manila-provisioner-env
+                  key: os_password
+            - name: "OS_AUTH_URL"
+              valueFrom:
+                secretKeyRef:
+                  name: manila-provisioner-env
+                  key: os_auth_url
+            - name: "OS_DOMAIN_NAME"
+              valueFrom:
+                secretKeyRef:
+                  name: manila-provisioner-env
+                  key: os_domain_name
+            - name: "OS_TENANT_NAME"
+              valueFrom:
+                secretKeyRef:
+                  name: manila-provisioner-env
+                  key: os_tenant_name
+----
+
+[[usage]]
+== Usage
+
+After the provisioner is running you can provision PersistentVolumes by using a
```

Comment:
Suggest changing this to "After the provisioner is running, you can provision PVs using a PVC and the corresponding StorageClass:"